meta-lmp-base/conf/distro/lmp.conf

```diff
@@ -13,6 +13,7 @@ OSTREE_KERNEL ?= "${@oe.utils.conditional('KERNEL_IMAGETYPE', 'fitImage', '${KER
 OSTREE_KERNEL_ARGS_COMMON ?= "root=LABEL=otaroot rootfstype=ext4"
 OSTREE_KERNEL_ARGS ?= "${OSTREE_KERNEL_ARGS_COMMON}"
 OSTREE_SPLIT_BOOT ?= "0"
+OSTREE_DEPLOY_USR_OSTREE_BOOT ?= "0"
```

Comment:
it seems to me that this new variable `OSTREE_DEPLOY_USR_OSTREE_BOOT` is implicit in the one `INITRAMFS_RECOVERY_IMAGE` introduced in the previous patch

Reply:
Yes, I decided to move this to a variable instead (and have it set at the image level) to avoid having multiple recipes/packages shipping the same file, which would fail at the package install/do_rootfs process.

Reply:
Oh sure, I decided to use a new flag because this is also going to be used for other binaries, such as firmware related ones, so not really specific to the recovery image, that was just the first user for this.

Having OSTREE_DEPLOY_USR_OSTREE_BOOT = 1 without OSTREE_DEPLOY_USR_OSTREE_BOOT won't cause any bad side effect, as there won't be any file to copy / deploy.

Reply:
> Oh sure, I decided to use a new flag because this is also going to be used for other binaries, such as firmware related ones, so not really specific to the recovery image, that was just the first user for this.

makes sense for me now